app/src/src/components/views/LikedUserView.jsx

```diff
@@ -0,0 +1,21 @@
+import { Text, View, StyleSheet, Image } from 'react-native';
+
+const LikedUserView = ({ first_name, profile_photo }) => {
+  return (
+    <>
+      <View style={styles.ListBegin}>
+        <Text style={styles.text}>{first_name}</Text>
+        <Image style={styles.image} source={profile_photo}></Image>
+      </View>
+    </>
```

Comment:
We do not need to wrap this in a react fragment (<></>). We can simply just return the View component